obsidian.d.ts

```diff
@@ -1118,74 +3530,189 @@ export interface DataWriteOptions {
  * await sleep(500);
  * debounced('World, hello'); // this will be printed to the console.
  * ```
+ *
  * @public
  */
 export function debounce<T extends unknown[], V>(cb: (...args: [...T]) => V, timeout?: number, resetTimer?: boolean): Debouncer<T, V>;
 
-/** @public */
+/**
+ * A debouncer wrapper for a function.
+ *
+ * @typeParam T - The type of the arguments of the function to debounce.
+ * @typeParam V - The type of the return value of the function to debounce.
+ *
+ * @public
+ */
 export interface Debouncer<T extends unknown[], V> {
-    /** @public */
+    /**
+     * Call the debounced function.
+     *
+     * @param args - The arguments to pass to the function.
+     * @returns The debouncer.
+     *
+     * @example
+     * ```ts
+     * debouncer('foo');
+     * ```
+     * @public
+     */
     (...args: [...T]): this;
     /**
      * Cancel any pending debounced function call.
+     *
+     * @returns The debouncer.
+     *
+     * @example
+     * ```ts
+     * debouncer.cancel();
+     * ```
      * @public
      */
     cancel(): this;
     /**
      * If there is any pending function call, clear the timer and call the function immediately.
+     *
+     * @returns The return value of the function or `void` if the are no pending function calls.
+     *
+     * @example
+     * ```ts
+     * debouncer.run();
+     * ```
      * @public
      */
     run(): V | void;
 }
 
 /**
  * Manually trigger a tooltip that will appear over the provided element.
- *
  * To display a tooltip on hover, use {@link setTooltip} instead.
+ *
+ * @param newTargetEl - The element to display the tooltip over.
+ * @param content - The content of the tooltip.
+ * @param options - The options for the tooltip.
+ *
+ * @example
+ * ```ts
+ * displayTooltip(document.body, 'foo');
+ * ```
+ *
  * @public
  */
 export function displayTooltip(newTargetEl: HTMLElement, content: string | DocumentFragment, options?: TooltipOptions): void;
 
 /**
+ * Dropdown component
+ *
  * @public
  */
 export class DropdownComponent extends ValueComponent<string> {
     /**
+     * The select element
```

Comment:
Maybe something more like:
The HTML element representation of this dropdown

Reply:
Thanks